src/transformers/modeling_bart.py

```diff
@@ -792,11 +823,6 @@ def fill_with_neg_inf(t):
     return t.float().fill_(float("-inf")).type_as(t)
 
 
-def _filter_out_falsey_values(tup) -> Tuple:
```

Comment:
This function is not necessary anymore since the empty/none things are filtered in the encoder/decoder.